specification/httpxe/src/main/scripts/org/kaazing/specification/httpxe/responses/header.in.connection.header.but.not.body/request.rpt

```diff
@@ -0,0 +1,14 @@
+connect http://localhost:8000/path
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write flush
+
+read status "200" "OK"
+read header "Content-Type" "application/x-message-http"
+read header "Connection" "keep-alive"
+
+read "101 Switching Protocols HTTP/1.1\r\n"
+read "Content-Type: text/html\r\n"
+

```

Comment:
This is showing that some headers are not moved to the response body. In this case there is a Connection header in the 200 response but it does not move to the response body. (Where the 101 response is)